setup.cfg

```diff
@@ -70,6 +71,3 @@ warn_unused_ignores = False
 
 [mypy-urllib3.*]
 ignore_missing_imports = True
-
-[mypy-apscheduler.*]
-ignore_missing_imports = True
```

Comment:
Why is this removed? I ran `mypy telegram/` on my machine and now it complains about missing type hints / stubs

Reply:
mh, interesting that it doesn't complain during pre-commit … anyway, I reverted :)